alws/models.py

```diff
@@ -48,6 +102,22 @@ class Platform(Base):
     repos = relationship('Repository', secondary=PlatformRepo)
 
 
+class Distribution(Base):
+
+    __tablename__ = 'distributions'
+
+    id = sqlalchemy.Column(sqlalchemy.Integer, primary_key=True)
+    name = sqlalchemy.Column(
+        sqlalchemy.Text,
+        nullable=False,
+        unique=True,
+        index=True
+    )
+    platforms = relationship('Platform', secondary=PlatformDependency)
+    repositories = relationship('Repository', secondary=DistributionRepositories)
+    packages = relationship('Build', secondary=DistributionPackages)
```

Comment:
I believe, this should be either relationship with `Packages`, or should be named `builds`